docs/UserGuide.md

```diff
@@ -156,7 +149,25 @@ Format: `findRoom`
 
 * Finds the room number of least value that can be safely used for accommodation
 
-### 3.8 Add a task to a room: `addtask`
+### 3.8 Allocate patient to room or edit room: `editroom`
+
+Allocate a patient to a room or edit an exisiting room in the application.
```

Comment:
Allocates*, edits* and existing*. Can change only when we do our massive UG clean up also.

Reply:
Since i have to change other parts, I think i will just update it in case I forget about it in the future